hbase-server/src/main/java/org/apache/hadoop/hbase/master/normalizer/RegionNormalizerWorker.java

```diff
@@ -199,6 +207,21 @@ public void run() {
     }
   }
 
+  private void sleepToNextHour() {
```

Comment:
nit: Add an info log saying offpeak hours is configured and we'll wait for offpeak hours to continue normalising.